cmd/flags_assigners.go

```diff
@@ -379,6 +379,37 @@ func assignCreateGroupOptions(cmd *cobra.Command) (*gitlab.CreateGroupOptions, e
 	return &opts, nil
 }
 
+// assignAddProjectMemberOptions assigns the flags' values to gitlab.AddProjectMemberOptions fields.
+// If a flag's default value is not changed by the caller,
+// it's value will not be assigned to the associated gitlab.AddProjectMemberOptions field.
+func assignAddProjectMemberOptions(cmd *cobra.Command) (*gitlab.AddProjectMemberOptions, error) {
+	var opts gitlab.AddProjectMemberOptions
+
+	if f := cmd.Flag("access-level"); f != nil {
```

Comment:
I think this should be checked using`cmd.Flag("access-level").Changed` since the flag is only meant to be assigned for `new member` command.